spec.md

```diff
@@ -173,6 +173,86 @@ minimally trusted storage servers is required.
+
+The initial W.A.S. Authorization Profile uses the following specifications.
+
+1. Identity (for controllers or clients/agents): [DID 1.1](https://www.w3.org/TR/did-1.1/)
+   or [CID 1.0](https://www.w3.org/TR/cid-1.0/) identifiers.
+2. Capability data model: [Authorization Capabilities for Linked Data v0.3](https://w3c-ccg.github.io/zcap-spec/)
+3. Protocol for obtaining authorization: Out of scope (implementers are encouraged
+   to use VC-API, OpenId4VP, or GNAP, as appropriate)
+4. Proof of Possession / authorization invocation: [HTTP Signatures (Cavage
+   draft 12)](https://datatracker.ietf.org/doc/html/draft-cavage-http-signatures)
+5. Access Control / Policy language data model: TBD
+
+#### Space `controller` and the Root of Trust
+
+Space `controller`s MUST be in the form of a [DID](https://www.w3.org/TR/did-1.1/)
+or a [CID](https://www.w3.org/TR/cid-1.0/) identifier.
+
+For minimal compatibility, all WAS implementations MUST support the
+[`did:key` DID Method](https://w3c-ccg.github.io/did-key-spec/), using the
+Multikey encoding of `Ed25519` elliptic curve keys, as specified in the
+[Multikey section of the CID spec](https://www.w3.org/TR/cid-1.0/#Multikey)
+as the space `controller`.
+
+When a space is created via an HTTP [POST](#http-api-post-spaces) or
+[PUT](#http-api-put-space-space_id) operation, the controller for that space
+is set, either implicitly or explicitly. 
+
+Implicitly, if no `controller` is specified in the PUT or POST create space
+request, the server MUST determine and set the `controller` from the corresponding
+authorization headers of the request (for example, from the `Authorization` header
+when using HTTP Signatures).
+
+Explicitly, if a client specifies the `controller` as part of the payload
+of the PUT or POST create space request, the server MUST check that the methods
+(key IDs) used in the headers are authorized in the `capabilityInvocation`
+section of the `controller`'s DID (or CID) document.
+
+See below in the [HTTP POST](#http-api-post-spaces) sections for examples of
+`controller` determination and verification.
+
+Conceptually, the space's controller serves as the root of trust and authorization
+for any operations on the space or its collections or resources.
+That is, any operation requiring an authorization MUST provide a chain of proof
+all the way to the space controller, by one of the following:
+
+1. Direct: Provide a root capability invoked directly by the controller, or
+2. Delegated: Invoke a capability delegated to some other agent by the controller, or
+3. Matching Policy: (if using any kind of access control policy mechanism) Match
+   an authorization policy specified in the `link` property of the space. This
+   resource is related to the space controller because initially, it can only be
+   modified either by the controller or an authorized party delegated to by the
+   controller.
+
+#### Performing Authorized API Calls
+
+Clients performing 
```

Comment:
This section seems a WIP

Reply:
maybe leave it for a future PR

Reply:
Of course it's a WIP :) This thing's still in Draft mode, and I'm still adding to it.

Reply:
> maybe leave it for a future PR

ohh ok, sure!